src/pages/splash/splash.less

```diff
@@ -248,6 +248,18 @@
   }
 
   /**
+   * Maker Party Promo
+   */
+  #mp-promo {
+    .promo {
+      .promoMixin(@vikingYellow);
```

Comment:
My assumption is there could be more than one, a reusable `<Promo>` component comes in handy, but I could be wrong!

(not sure who to flag...)@xmatthewx @vazquez ? :hamster: